src/keys/bip39.rs

```diff
@@ -36,33 +38,33 @@ pub enum Error {
 }
 
 /// Reference to a normalized (unicode NFKD) mnemonic.
-#[derive(Clone, Copy)]
-pub struct MnemonicRef<'a>(&'a str);
+pub struct MnemonicRef(str);
```

Comment:
`#[repr(transparent)]`?

Reply:
This is needed for all types you are transmuting btw. Otherwise it really is unsafe.

Reply:
Exactly. Done.